jobs/config.json

```diff
@@ -2586,7 +2586,7 @@
     ],
     "scenario": "kubernetes_e2e",
     "sigOwners": [
-      "UNKNOWN"
+      "sig-scale"
```

Comment:
s/sig-scale/sig-scalability/
Renamed for other jobs in https://github.com/kubernetes/test-infra/pull/5359.

Reply:
thanks, I didn't want to slime that into this PR, I'll rebase